Dockerfile

```diff
@@ -1,14 +1,18 @@
-FROM python:2.7
+FROM python:2.7.12-alpine
+
+RUN apk add --update --no-cache alpine-sdk wget git
```

Comment:
@nivong: before you take out wget you would have to modify the Dockerfile like this:

-RUN cd /tmp && wget http://pgoapi.com/pgoencrypt.tar.gz \
-      && tar zxvf pgoencrypt.tar.gz \
+ADD http://pgoapi.com/pgoencrypt.tar.gz /tmp/
+RUN cd /tmp && tar zxvf pgoencrypt.tar.gz \

as for git i don't know if is used